app/assets/stylesheets/addons/_directional-values.scss

```diff
@@ -1,22 +1,31 @@
-// directional-property mixins are shorthands
-// for writing properties like the following
-//
-// @include margin(null 0 10px);
-// ------
-// margin-right: 0;
-// margin-bottom: 10px;
-// margin-left: 0;
-//
-// - or -
-//
-// @include border-style(dotted null);
-// ------
-// border-top-style: dotted;
-// border-bottom-style: dotted;
-//
-// ------
-//
-// Note: You can also use false instead of null
+@charset "UTF-8";
+
+/// directional-property mixins are shorthands
+/// for writing properties like the following
+///
+/// @ignore You can also use `false` instead of `null`.
+///
+/// @param {List} $vals
+///   List of directional values
+///
+/// @example scss - Usage
+///   .wrapper {
+///     @include border-style(dotted null);
+///     @include margin(null 0 10px);
+///   }
+///
+/// @example css - CSS Output
+///   .wrapper {
+///     border-bottom-style: dotted;
+///     border-top-style: dotted;
+///     margin-bottom: 10px;
+///     margin-left: 0;
+///     margin-right: 0;
+///   }
+///
+/// @require {function} contains-falsy
```

Comment:
I guess I was reading the above as documenting `collapse-directionals()`. It's hard to tell what we're documenting here since both `border-style` and `margin` are below and also have the `@require {function} contains-falsy` line.